pkg/metrics/metrics.go

```diff
@@ -235,29 +263,6 @@ func isEndpointsValid(nodes *v1.NodeList, endpoints *v1.Endpoints) bool {
 	return true
 }
 
-// updateServiceMonitors patches the metrics Service Monitor to include required fields to display node graphs on the
-// OpenShift console. Console graph queries require metrics endpoint target name to be node name, however
-// windows_exporter returns node IP. We replace the target name by adding `replace` action field to the ServiceMonitor
-// object that replaces node IP to node name as the metrics endpoint target.
-func updateServiceMonitors(cfg *rest.Config, namespace string) error {
```

Comment:
Where is this being done now?

Reply:
This is being done now in the manifest for serviceMonitor in manifests/serviceMonitor.yaml